source/includes/fundamentals/code-snippets/CRUD/bulkOps.go

```diff
@@ -52,13 +53,15 @@ func main() {
 
 	fmt.Println("\nInsertOneModel:\n")
 	{
+		// Creates write models that specify insert operations
```

Comment:
S: I followed this format when writing comments for the Java write models, and kinda avoided using "write models" in the comments themselves
```suggestion
		// Creates instructions to insert documents describing books
```